### DISCUSSION TITLE
Monitoring active calls

### DISCUSSION THREAD

**Discussion by MrLyu49** [mikopbx/Core#760] [General ( Обо всём )]
В веб-интерфейсе Мико на вкладке Перезагрузка есть замечательная фишка - мониторинг активных вызовов. Очень бы хотелось пользоваться этой функцией и без мысли о перезагрузке, т.е. вынести ее отдельно. И в частности, дать ограниченный доступ к этой функции через модуль Управление доступом в систему. 

Это нужно, чтобы начальство могло контролировать звонки подчиненных и оперативно подключаться к некоторым разговорам. Сейчас Мико тоже дает возможность оперативного подключения (функции суфлера), да только при удаленной работе сотрудников явно не хватает мониторинга!

Итак, хотелось бы дать начальству доступ к мониторингу (без Перезагрузки, естественно). Тогда можно было бы в нужный момент подключиться к разговору, нажав на своем телефоне кнопочку с одной из запрограммированных функций суфлера для интересующего подчиненного.

При этом мониторинге показывать не вообще всех, а только тех сотрудников, кого включили в список контроля Истории вызовов.

При Активных вызовах дать кнопочку(-и) подключения к разговору (в разных режимах суфлера). В истории можно прослушать запись - а в активных надо подключаться онлайн!

И хорошо бы совместить на одной вкладке интерфейса Историю вызовов и Активные вызовы. Вверху 1-й странички Истории высвечиваются ярким цветом активные вызовы, по мере завершения постепенно переходя в собственно Историю.

**Comment by boffart** [comment]
Перевел задачу в обсуждения. 
Мы обсуждали в команде создание аналога FOP 
https://www.fop2.com/
пока нет ресурсов на создание нового продукта. Мб в будущем займемся. 

Приоритет отдаем разработке "ядра" АТС.
